sound/soc/sof/intel/hda-bus.c

```diff
@@ -70,9 +70,14 @@ void sof_hda_bus_init(struct snd_sof_dev *sdev, struct device *dev)
 {
 	struct hdac_bus *bus = sof_to_bus(sdev);
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA_LINK)
```

Comment:
grammatical error in the commit message "but for some tests, "we" need" or "but some tests"

Reply:
yes, fixed.